php/class-fm-bylines-cli.php

```diff
@@ -0,0 +1,134 @@
+					update_post_meta( $byline_id, 'fm_bylines_names', array(
+						'first_name' => ! empty( $cap_meta['cap-first_name'][0] ) ? $cap_meta['cap-first_name'][0] : '',
+						'last_name' => ! empty( $cap_meta['cap-first_name'][0] ) ? $cap_meta['cap-last_name'][0] : '',
+					) );
+
+					update_post_meta( $byline_id, 'fm_bylines_contact_info', array(
+						'email' => ! empty( $cap_meta['cap-user_email'][0] ) ? $cap_meta['cap-user_email'][0] : '',
+						'website' => ! empty( $cap_meta['cap-website'][0] ) ? $cap_meta['cap-website'][0] : '',
+						'twitter' => ! empty( $cap_meta['cap-twitter'][0] ) ? $cap_meta['cap-twitter'][0] : '',
+						'linkedin' => ! empty( $cap_meta['cap-linkedin'][0] ) ? $cap_meta['cap-linkedin'][0] : '',
+					) );
+
+					update_post_meta( $byline_id, 'fm_bylines_about', array(
+						'bio' => ! empty( $cap_meta['cap-description'][0] ) ? $cap_meta['cap-description'][0] : '',
+						'short-bio' => ! empty( $cap_meta['cap-shortbio'][0] ) ? $cap_meta['cap-shortbio'][0] : '',
+					) );
+
+					if ( ! empty( $cap_meta['cap-linked_account'][0] ) ) {
+						$mapped_user = get_user_by( 'slug', $cap_meta['cap-linked_account'][0] );
+						if ( ! empty( $mapped_user ) ) {
+							update_post_meta( $byline_id, 'fm_bylines_user_mapping', $mapped_user->ID );
+						}
+					}
+				} else {
+					$byline_id = $byline_post[0]->ID;
+				}
+
+				do_action( 'fm_bylines_cli_post_cap_migration', $byline_id, $cap_meta, $cap_term, $cap_post[0] );
+
+				// Migrate the legacy CAP metadata and store in case custom meta fields were missed.
+
+				// Clean legacy data. Don't delete by default. This will migrate the featured image as well.
+				if ( apply_filters( 'fm_bylines_cli_clean_legacy_cap_data', false ) ) {
+					//We are just flipping the post id here.
+					global $wpdb;
+					$wpdb->update( $wpdb->postmeta, array( 'post_id' => $byline_id ), array( 'post_id' => $cap_post[0]->ID ), array( '%d' ), array( '%d' ) );
+				} else {
+					// Copy cap meta
+					foreach ( $cap_meta as $key => $value ) {
+						update_post_meta( $byline_id, $key, $value[0] );
+					}
+				}
+
+				// Update posts so byline authors reflect CAP authors
+				$objects = get_objects_in_term( $cap_term->term_id, $coauthors_plus->coauthor_taxonomy );
+				foreach ( $objects as $object_id ) {
+					if ( get_post_type( $object_id ) !== $coauthors_plus->guest_authors->post_type ) {
+						$current_bylines = empty( get_post_meta( $byline_id, 'fm_bylines_author', true ) ) ? array() : get_post_meta( $byline_id, 'fm_bylines_author', true );
+						$new_byline_entry = array(
+							'byline_id' => $byline_id,
+							'fm_byline_type' => $coauthors_plus->coauthor_taxonomy,
+						);
+						$current_bylines[] = $new_byline_entry;
+						update_post_meta( $byline_id, 'fm_bylines_' . $coauthors_plus->coauthor_taxonomy, $current_bylines );
```

Comment:
Cap is only authors. If for some reason they have changed this, this accounts for it here.